src/app/lib/layoutGrid.js

```diff
@@ -42,41 +42,33 @@ export const layoutWrapperWithoutGrid = css`
 export const layoutGridWrapper = css`
   display: -ms-grid;
   display: grid;
+  margin: 0 auto;
+  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+
   @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    grid-gap: ${GEL_GUTTER_BELOW_600PX};
+    grid-column-gap: ${GEL_GUTTER_BELOW_600PX};
   }
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    grid-gap: ${GEL_GUTTER_ABOVE_600PX};
-  }
-  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    padding: 0 ${GEL_MARGIN_BELOW_400PX};
-  }
-  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-    padding: 0 ${GEL_MARGIN_ABOVE_400PX};
+    grid-column-gap: ${GEL_GUTTER_ABOVE_600PX};
   }
   @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
     -ms-grid-columns: (1fr)[6]; // prettier-ignore
-    grid-template-columns: repeat(6, 1fr);
+    grid-template-columns: 0 repeat(6, 1fr) 0;
   }
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
     -ms-grid-columns: 1fr (minmax(0, ${group4ColWidth}))[8] 1fr; // prettier-ignore
     grid-template-columns: 1fr repeat(8, minmax(0, ${group4ColWidth})) 1fr;
```

Comment:
Feels to me that if we are doing the work to go `full bleed` for some of the groups that we should we it for all of them and therefore resolve https://github.com/bbc/simorgh/issues/841 in the process

Reply:
Do we have UX designs for full bleed on all groups? I think we should leave it for now untill we have a better idea, also because that will make this PR bigger

Reply:
^ @jimjohnsonrollings agrees to leave it for now